tei/r2mn.0.xml

```diff
@@ -1001,8 +1001,8 @@
                 <l> Wen denn?</l>
                 <pb n="141"/>
               </sp>
-              <sp who="#guhli">
-                <speaker>GUHLI.</speaker>
+              <sp who="#gurli">
+                <speaker>GURLI.</speaker>
```

Comment:
```suggestion
                <speaker>GUHLI.</speaker>
```

Es wäre für uns relativ wichtig, dass sich der Text in den speaker-Tags nicht ändert (weil das ungefähr der gedruckten Ausgabe entspricht, selbst wenn da Fehler drin sind).

Reply:
@leaduempelmann Wenn Speaker-Tags Fehler ausweisen, wird das in `<listPerson>` abgefangen. In diesem Fall sieht das z.B. so aus:
```
<person xml:id="gurli">
<persName>GURLI</persName>
<persName xml:id="guhli">GUHLI</persName>
</person>
```

Reply:
Danke für die guten Hinweise. Ich hab mich auch gefragt, ob das in der Ausgabe so steht bzw. ob das mit den Fehlern dann gefunden wird.